Source/Core/DolphinWX/ISOFile.cpp

```diff
@@ -129,7 +131,14 @@ GameListItem::GameListItem(const std::string& _rFileName)
 		ini.GetIfExists("EmuState", "EmulationIssues", &m_issues);
 	}
 
-	if (!m_pImage.empty())
+	if (IsElfOrDol())
+	{
+		m_Bitmap.LoadFile(StrToWxStr(File::GetThemeDir(SConfig::GetInstance().theme_name)) + "nobanner.png", wxBITMAP_TYPE_PNG);
+		m_Valid = true;
+		m_FileSize = File::GetSize(_rFileName);
+		m_Platform = DiscIO::IVolume::ELF_DOL;
+	}
+	else if (!m_pImage.empty())
```

Comment:
Why `else if`? This could just be a separate `if` statement, then the line above for loading `nobanner.png` will be unnecessary.